LoRaEngine/LoraKeysManagerFacade/DeviceRegistryMode.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System.Runtime.Serialization;
+
+    internal enum DeviceRegistryMode
+    {
+        IoTHub,
+        IoTCentral
```

Comment:
More than IoT Central, it's DPS that uses the mechanism. So what is specific to IoT Central that you have to do here and not just related to DPS?
And this does apply to the URL and other elements like this one. I may point some out as well.

Reply:
So if there is no better way to have the IoT Central API, then I'm fine with IoT Central here.